api/jsonapi_data_test.go

```diff
@@ -366,7 +369,8 @@ const walletBalanceJSONResponse = `{
     "TZEC": {
         "confirmed": "0",
 				"currency": {
-						"code": "TZEC",
+                        "BlockTime": 600000000000,
+                        "code": "TZEC",
```

Comment:
I don't think we should put blocktime into the persisted schema. I think leaving it as a runtime datapoint is sufficient. Let's ensure this data doesn't leak into our various outputs.